keyboards/planck/keymaps/tk/config.h

```diff
@@ -0,0 +1,163 @@
+#pragma once
+
+#define ENCODER_RESOLUTION 4    // dependent on rotary encoder
+
+#define TAP_CODE_DELAY 10       // delay between key press and release on `tapcode(kc)`
+
+#define TAPPING_TERM 200        // mod tap tap-window
+
+// oneshot keys --
+
+#define ONESHOT_TAP_TOGGLE 25   // taps to toggle a one-shot layer/key persistence
+
+#define ONESHOT_TIMEOUT 500     // one-shot layer/key timeout
+
+// dynamic macros --
+
+// you can store HALF of this number of keypresses in both macros together
+#define DYNAMIC_MACRO_SIZE 256
+
+// mouse keys --
+
+#define MK_3_SPEED
+
+// mouse speeds
+
+// cursor offset per movement
+#define MK_C_OFFSET_1 4
+#define MK_C_OFFSET_2 16
+
+// time between cursor movements (ms)
+#define MK_C_INTERVAL_1 16  // 16 ms = 60 Hz refresh rate
+#define MK_C_INTERVAL_2 16
+
+// scroll speeds
+
+// scroll steps per scroll action
+#define MK_W_OFFSET_1 1
+#define MK_W_OFFSET_2 4
+
+// time between scroll steps (ms)
+#define MK_W_INTERVAL_1 16
+#define MK_W_INTERVAL_2 16
+
+
+// audio --
+
+#define AUDIO_CLICKY
+#define AUDIO_CLICKY_FREQ_RANDOMNESS 1
+
+#ifdef AUDIO_ENABLE
+
+    #undef  TEMPO_DEFAULT       // set a custom tempo
+    #define TEMPO_DEFAULT 255   // smaller is faster; max = 255
+
+    #define T__NOTE(note) MUSICAL_NOTE(note, 2) // 32nd-note
+    #define TD_NOTE(note) MUSICAL_NOTE(note, 3) // 48th-note
+    #define X__NOTE(note) MUSICAL_NOTE(note, 1) // 64th-note
+
+    #define NOTE_A5SEMI         905.79f
+    #define NOTE_AS5SEMI        959.65f
+    #define NOTE_B5SEMI         1016.71f
+
+    // Mario songs
+
+    #define MARIO_FIREBALL      T__NOTE(_G4), T__NOTE(_G5), T__NOTE(_G6), T__NOTE(_G7)
```

Comment:
As much as we'd love to accept these, it would be problematic at best. 

However, you can add them to a `user_song_list.h` file, and they will be automatically included, and can be used and not included in the repo.